### DISCUSSION TITLE
Weird channel naming -- channel designated as "8" appears in plex as "C8.152.tunarr.com"

### DISCUSSION THREAD

**Discussion by princinv** [chrisbenincasa/tunarr#930] [Questions & Help] [answered]
Anybody know how to make this a simple "8"? i have no idea where the 152 comes from but it is different for each one, or why it is appended with tunarr.com. 

I am running it on a docker node with plex on the manager but they are directly connected over ethernet. 

I cant remember exactly what it was before i moved to a node but it still wasnt a simple "8", it included "tunarr" at least.

Not a huge deal it just makes me have to adapt the naming of my IPTV channels and looks cluttered when there is no channel icon.

**Comment by chrisbenincasa** [comment, marked as answer]
This is an unfortunate workaround for this issue in Plex: https://forums.plex.tv/t/dots-in-xmltv-channel-ids-cause-inaccurate-live-tv-guide-listings/889712

I would be willing to expose this as an option though so that non-Plex client users don't have to suffer with the workaround

**Reply by chrisbenincasa** [reply]
This bug describes the issue more: https://github.com/chrisbenincasa/tunarr/issues/692

**Reply by princinv** [reply]
that makes sense. i was previously having the issue of merged channels as well. id much rather have this. 

thanks for the quick reply, and good work on the app.
